Assets/Plugins/Android/login.java

```diff
@@ -0,0 +1,121 @@
+/*
```

Comment:
What is the origin of this file? If it is something we made, then maybe there's some stylistic things to consider, but I also don't know Java conventions well.

Reply:
I wrote this from scratch. And I am not familiar with any java conventions